### ISSUE TITLE
Add support for Channel 9 video

### ISSUE THREAD

**Issue by paul90** [fedwiki/wiki-plugin-video#9]
It would be nice to be able to embed MSDN Channel 9 videos

**Comment by WardCunningham** [comment]
Poking around a bit I found this example.

http://channel9.msdn.com/Series/Microsoft-Research-Luminaries/Eric-Horvitz-on-the-new-era-of-Artificial-Intelligence#time=0h12m40s:paused

Which suggests this as an embed code.

```
<iframe src="//channel9.msdn.com/Series/Microsoft-Research-Luminaries/Eric-Horvitz-on-the-new-era-of-Artificial-Intelligence/player" width="960" height="540" allowFullScreen frameBorder="0"></iframe>
```

Is there enough consistency in these to be useful for Channel 9 in general?

**Comment by san650** [comment]
I've created a simple implementation to support channel 9 videos, I think it covers most channel 9 videos. Please let me know your thoughts

**Comment by paul90** [comment]
Closed by #12 

Published in + wiki-plugin-video@0.2.4
